Don't grow the top of the glyph bounding box by a 64th.

CPDF_CIDFont::GetCharBBox() is the only caller of CFX_Face::GetCharBBox(),
so this made composite fonts measure glyphs taller than simple fonts, which
get their bounding box from CFX_Face::GetGlyphBBox() directly.

diff -ru pdfium.orig/core/fxge/cfx_face.cpp pdfium/core/fxge/cfx_face.cpp
--- pdfium.orig/core/fxge/cfx_face.cpp
+++ pdfium/core/fxge/cfx_face.cpp
@@ -68,10 +68,6 @@
 
 constexpr int kMaxGlyphDimension = 2048;
 
-// Boundary value to avoid integer overflow when adding 1/64th of the value.
-constexpr int kMaxRectTop = 2114445437;
-
-
 int FTPosToCBoxInt(FT_Pos pos) {
   // Boundary values to avoid integer overflow when multiplied by 1000.
   static constexpr FT_Pos kMinCBox = -2147483;
@@ -1181,11 +1181,6 @@
     int err = FT_Load_Glyph(rec, glyph_index, FT_LOAD_NO_SCALE);
     if (err == 0) {
       rect = GetGlyphBBox();
-      if (rect.top <= kMaxRectTop) {
-        rect.top += rect.top / 64;
-      } else {
-        rect.top = std::numeric_limits<int>::max();
-      }
     }
   }
 #if defined(PDF_ENABLE_SKIA_TYPEFACE_CHECKS)

